express_webpack/index.html

```diff
@@ -154,6 +154,17 @@
         })
     }
 
+    function sendSelfNotification() {
+        OneSignal.push(function() {
+          OneSignal.sendSelfNotification('Title', 'message', 'https://localhost:4001/?app_id=90ab655e-f1b6-4a72-9b0e-69e842f24033', 'https://t3.ftcdn.net/jpg/03/08/73/34/360_F_308733458_QBzwMVu8ZzdGEp9Wwq1fAYaDgtP3UVwl.jpg', { test: 'foo' }, [{
```

Comment:
If we leave this new test function in here we should pull the `app_id` instead of hard coding one.